src/main/java/com/sintern/service/EncryptionServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package com.sintern.service;
+
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class EncryptionServiceImpl implements EncryptionService {
+
+    public EncryptionServiceImpl() {
+    }
+
+    public String encodePassword(String password){
+        BCryptPasswordEncoder passwordEncoder = new BCryptPasswordEncoder();
+        return passwordEncoder.encode(password);
+
```

Comment:
Does this improve readability? Delete redundant white spaces.

Reply:
Done